Fix release 20220908 docs (#1231)
objectiv/objectiv-analytics: bach/docs/source/example-notebooks/open-taxonomy.rst

When instantiating the model hub, global contexts are added using the name of the context without the word
'Context' and converted to 'snake_case' (the name of the context split before every capital letter and

> joined with an underscore), i.e. 'HttpContext' becomes 'http' and 'InputValueContext' becomes 'input_value' 
> when added to `global_contexts`:

Reviewer: ```suggestion
joined with an underscore), i.e. to add the HttpContext use 'http' and to add the InputValueContext use
'input_value':
```

All renders good, but I do have a suggestion for the phrasing actually. I think the suggested change makes it clearer that this conversion to snake_case is something that the user has to do.

Author: I like it, let's use your suggestion.
